packages/types/src/localization.ts

```diff
@@ -1165,15 +1164,26 @@ export type __internal_LocalizationResource = {
       formTitle: LocalizationValue<'apiKeyName'>;
       formHint: LocalizationValue;
       formButtonPrimary__revoke: LocalizationValue;
-    };
-    dates: {
-      lastUsed__seconds: LocalizationValue<'seconds'>;
-      lastUsed__minutes: LocalizationValue<'minutes'>;
-      lastUsed__hours: LocalizationValue<'hours'>;
-      lastUsed__days: LocalizationValue<'days'>;
-      lastUsed__months: LocalizationValue<'months'>;
-      lastUsed__years: LocalizationValue<'years'>;
-    };
```

Comment:
Could this be considered a breaking change? 🤔

Reply:
Hi @anagstef! This component is in early preview and you need to have your instance flagged for API keys access.  We also don't have the locales documented. It should be safe. Unless I'm missing something 🤔